Gemfile

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
+ruby '3.1.0'
 gem 'rspec'
+gem 'rubocop-gemfile', require: false
+gem 'simplecov', require: false, group: :test
 # gem "rails"
```

Comment:
You should remove lines with commented gems.

IMHO, this comment informs nothing after you generate the gemfile for the fisrt time.